### PR TITLE
Updates from Eris

### PR DESCRIPTION
**Pull request by kcho**

### CODE REVIEW

**Review by owenborders — APPROVED** [review]
Not a big deal but some parts of this can maybe be made less redundant with more functions or for loops.

File: ampscz_asana/lib/mri_count.py [hunk 1]
```diff
@@ -25,11 +25,11 @@ def count_and_make_it_available_for_dpdash(phoenix_paths: List[Path],
         # mriqc_dir everyday. This QC information is added to zip_df
```
Comment (nickckim) [inline ampscz_asana/lib/mri_count.py:25]: Typo in line 20: mathing → matching

File: qqc/pipeline.py [hunk 2]
```diff
@@ -121,15 +126,16 @@ def dicom_to_bids_QQC(args, **kwargs) -> None:
         config.read(args.config)
         for root, dirs, files in os.walk(qqc_input):
             for subdir in dirs:
-                if 't1w_mpr_nd' in subdir.lower():
+                if is_xa30(qqc_input):
                     # TODO add enhanced logic
```
Comment (nickckim) [inline qqc/pipeline.py:130]: Obsolete comment in line 130

File: scripts/move_extra_scans.py [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+    scope = ["https://spreadsheets.google.com/feeds",
+            "https://www.googleapis.com/auth/spreadsheets",
+            "https://www.googleapis.com/auth/drive.file",
+            "https://www.googleapis.com/auth/drive"]
+
+    json_loc = "/data/predict1/home/nick/json_keys/gspread_official_key.json"
+    creds = ServiceAccountCredentials.from_json_keyfile_name(json_loc, scope)
+    client = gspread.authorize(creds)
+
+    # Open the Google Spreadsheet using its name
```
Comment (nickckim) [inline scripts/move_extra_scans.py:92]: Looks good. get_gs_sheet and gs_sheet_to_df from amp_scz_mri_ss kcho branch could also be useful here.

**Review by nickckim — APPROVED** [review]

File: ampscz_asana/lib/mri_count.py (hunk 1 above)
Comment (kcho) [reply ampscz_asana/lib/mri_count.py:25]: [42afc63](https://github.com/AMP-SCZ/qqc/pull/52/commits/42afc63aeeaaf7713b65ab04c286069218cf0af0)

File: qqc/pipeline.py (hunk 2 above)
Comment (kcho) [reply qqc/pipeline.py]: [2a5844e](https://github.com/AMP-SCZ/qqc/pull/52/commits/2a5844ea5a60809656ba5ef88086e1ea517596b5)

File: scripts/move_extra_scans.py (hunk 3 above)
Comment (kcho) [reply scripts/move_extra_scans.py:92]: We should plan to merge `amp_scz_mri_ss` to `qqc.amp_scz_mri` soon.

**Pull request merged**